src/factors/flux/MixtureFluxPose2Pose2.jl

```diff
@@ -21,18 +20,24 @@ mutable struct PackedMixtureFluxPose2Pose2 <: PackedInferenceType
   specialSampler::String
 end
 
-function sampleMixtureFluxPose2Pose2( nfb::MixtureFluxPose2Pose2,
-                                      N::Int,
-                                      fmd::FactorMetadata,
-                                      legacy... )
+
+function IIF.getSample(cfo::CalcFactor{<:MixtureFluxPose2Pose2}, N::Int=1)
   #
+  nfb = cfo.factor
+  fmd = cfo.metadata
+
   # only calculate once after default value of 0
   if nfb.DT[] == 0
     # cache the time difference estimate
     nfb.DT[] = (getTimestamp(fmd.fullvariables[2]) - getTimestamp(fmd.fullvariables[1])).value * 1e-3
   end
 
-  return getSample(nfb.Zij, N)
+  cf_ = CalcFactor( cfo.factor.Zij, cfo.metadata, 0, length(cfo._legacyMeas), cfo._legacyMeas, cfo._legacyParams)
```

Comment:
this is fine for now, will do better with the CCW---CF make over.

Reply:
https://github.com/JuliaRobotics/IncrementalInference.jl/issues/1166